internal/zinc-apiinfo/src/main/scala/xsbt/api/APIUtil.scala

```diff
@@ -13,7 +13,10 @@ object APIUtil {
   val modifiersToByte = (m: Modifiers) => {
     import m._
     def x(b: Boolean, bit: Int) = if (b) 1 << bit else 0
-    (x(isAbstract, 0) | x(isOverride, 1) | x(isFinal, 2) | x(isSealed, 3) | x(isImplicit, 4) | x(isLazy, 5) | x(isMacro, 6)).toByte
+    (x(isAbstract, 0) | x(isOverride, 1) | x(isFinal, 2) | x(isSealed, 3) | x(isImplicit, 4) | x(
+      isLazy,
+      5
+    ) | x(isMacro, 6)).toByte
```

Comment:
See http://scalameta.org/scalafmt/#Infixapplications line breaks around infix operators are your responsibility, so in this case scalafmt won't insert a line break before or after `|`